cvmfs/sync_mediator.cc

```diff
@@ -765,6 +778,7 @@ void SyncMediator::PrintChangesetNotice(const ChangesetAction  action,
 
 void SyncMediator::AddFile(SharedPtr<SyncItem> entry) {
   PrintChangesetNotice(kAdd, entry->GetUnionPath());
+  IngestionSource *SyncItem = entry->CreateIngestionSource();
```

Comment:
This change and the one on line 824 are not very helpful. Also the name of the variable of type `IngestionSource*` is not compliant with the style guide. It should be `sync_item` or something more explanative.

I would remove any changes not related to the purpose of the PR...